install_config/web_console_customization.adoc

```diff
@@ -68,6 +68,51 @@ link:https://github.com/openshift/origin-web-console/tree/master/extensions/exam
 Origin] repository on GitHub.
 ====
 
+[[setting-extension-properties]]
+=== Setting Extension Properties
+
+Extension scripts allow you to override the default behavior of the web console
+and customize it for your needs. For example, extension scripts can be used to
+add your own company's branding or to add company-specific capabilities. A
+common use case for this is rebranding or white-labelling for different
+environments. You can use the same extension code, but provide settings that
+change the web console.
+
+If you have a specific extension, but want to use different text in it for each
+of the environments, you can can define the environment in the
+*_master-config.yaml_* file, and use the same extension script across
+environments. Pass settings from the *_master-config.yaml_* file to be used by
+the extension using the
+xref:../install_config/master_node_configuration.adoc#master-config-asset-config[`extensionProperties`
+mechanism]:
+
+====
+[source,yaml]
+----
+assetConfig:
+  extensionDevelopment: true
+  extensionProperties:
+    doc_url: https://docs.openshift.com
+    hello: world
+    test: thing
```

Comment:
Yeah, arbitrary key/value pairs can be used, I was just showing a quick example (since I was unimaginative and couldn't think of a better one)